docs/usage/generated-api.md

```diff
@@ -0,0 +1,199 @@
+<script setup lang="ts">
+const eventConfigFile = `event MyEvent = {
+    from: Server,
+    type: Reliable,
+    call: ManyAsync,
+    data: (Options: struct {
+        foo: string,
+        bar: u8,
+    }),
+}
+
+event AnotherEvent = {
+    from: Client,
+    type: Reliable,
+    call: SingleAsync,
+    data: (Foo: boolean, Bar: u8)
+}
+`
+
+const functConfigFile = `funct GetScore = {
+    call: Async,
+    args: struct {
+        roundId: string,
+        category: enum { HighScore, LowScore, AverageScore },
+    },
+    rets: u16,
+}
+`
+</script>
+
+# Using Your Generated Code
+
+After [generating code](./generation.md), you will gain access to Zap's generated API. While this API is fully typesafe, meaning it will not throw errors for invalid usage, your static analysis tooling will. To solve this:
```

Comment:
This sentence doesn't make much sense.

Reply:
This sentence _is_ valid English, but if we're already getting confused by it, it should probably be changed. What in particular is nonsensical about it? It reads fine to me.

Reply:
The way it mentions typesafety is confusing, and it also contradicts the homepage
![Brave Browser-cJyK2fA5@2x](https://github.com/user-attachments/assets/fa5321ff-eb2c-4577-b6ad-5440622b7535)

As zap _will_ throw errors for invalid data (e.g. think wrong enums, writechecks if enabled) and buffer methods will also force valid usage.

Reply:
Ah, I see. I pretty much just reformatted the existing sentence: "While Zap will not throw errors for invalid usage, your static analysis tooling will."

Should be easy enough to rewrite appropriately.

Reply:
I'm thinking of just removing that blurb and the proceeding 2 bullet points. Anyone using Zap is probably already aware of those options, and they aren't really Zap specific for us to even address. Thoughts?

Reply:
Yeah - I'd hope they'd already know too. That's fine with me.